documentation/en/mining-lotus-seal-worker.md

```diff
@@ -12,39 +12,13 @@ IPFS_GATEWAY="https://proof-parameters.s3.cn-south-1.jdcloud-oss.com/ipfs/"
 
 ## Get Started
 
-Make sure that the `lotus-seal-worker` is installed by running:
+Make sure that the `lotus-seal-worker` is compiled and installed by running:
 
 ```sh
 make lotus-seal-worker
 ```
 
-## Running Alongside Storage Miner
```

Comment:
why would you delete this section?

Reply:
Because its in here 2 times :)

https://github.com/filecoin-project/lotus/pull/1961/files#diff-b14052ef2cf2486e7c90569e7bc77b21R43-R81

Reply:
Also running `lotus-seal-worker run` generates an error (`--address is required`)